addons/crm/models/crm_team_member.py

```diff
@@ -50,163 +70,105 @@ def _constrains_assignment_domain(self):
                     user=member.user_id.name, team=member.crm_team_id.name
                 ))
 
-    def _get_lead_month_domain(self):
-        limit_date = fields.Datetime.now() - datetime.timedelta(days=30)
-        return [
-            ('user_id', '=', self.user_id.id),
-            ('team_id', '=', self.crm_team_id.id),
-            ('date_open', '>=', limit_date),
-        ]
-
     # ------------------------------------------------------------
     # LEAD ASSIGNMENT
     # ------------------------------------------------------------
 
-    def _assign_and_convert_leads(self, work_days=1):
+    def _assign_and_convert_leads(self, force_quota=False):
```

Comment:
Note that in this new implementation, it is more a per-team assign than a per-member. Previously we made a population of members on which we have to assign leads, based on weight. Now it is more running on teams, take available members, and give them leads.

Not sure there is still something that is really member-based, can't we move the method on the "crm.team" model instead ?